File: select.js
const { Client } = require('pg')
const client     = new Client({database:"test", user:'aaaa', password:'aaaa', host:'localhost'})
client.connect()

client.query
 ('SELECT * from test',
  function(err, res)
   {console.log(err ? err.stack : res)
    client.end()
   }
 )

File: select.py
#!/usr/bin/python
import pgdb

C = pgdb.Connection(host='localhost', user='aaaa', password='aaaa', database='test')
c = C.cursor()

c.execute( "select * from test" )

for firstname, lastname in c.fetchall() :
  print( firstname, lastname )

c.close()
